l10n/en/firefox/more.ftl

```diff
@@ -23,9 +23,7 @@ choose-which-firefox = Choose which { -brand-name-firefox-browser } to download
 firefox-more-firefox-chromebook = { -brand-name-firefox-browser } for { -brand-name-chromebook }
 firefox-more-while-on-chromebook = While a { -brand-name-chromebook } already has { -brand-name-chrome } installed, downloading and using { -brand-name-firefox } as your go-to browser provides you with a few benefits
 firefox-more-firefox-quantum = { -brand-name-firefox-quantum }
-firefox-more-quantum-was-revolution ={ -brand-name-firefox-quantum } was a revolution in { -brand-name-firefox } development. In 2017, we created a new, lightning fast browser that constantly improves. { -brand-name-firefox-quantum } is the { -brand-name-firefox-browser }.
-firefox-more-little-book = Little Book of Privacy
-firefox-more-you-can-reclaim = You can reclaim your privacy and gain back control over your very own internet experience! It’s actually surprisingly easy. Everyone can do it.
+firefox-more-quantum-was-revolution = { -brand-name-firefox-quantum } was a revolution in { -brand-name-firefox } development. In 2017, we created a new, lightning fast browser that constantly improves. { -brand-name-firefox-quantum } is the { -brand-name-firefox-browser }.
```

Comment:
**note**: the `firefox-more-quantum-was-revolution = { -brand-name-` is just a whitespace fix.